book/chrome.md

```diff
@@ -869,12 +970,17 @@ get a little lost and forget what web page you're looking at. An
 address bar that shows the current URL would help a lot.
 
 ``` {.python}
-class Browser:
-    def paint_chrome(self):
+class Chrome:
+    def paint(self):
+        # ...
+        left_bar = addressbar_left + self.padding
+        top_bar = addressbar_top + self.padding
         # ...
-        cmds.append(DrawOutline(40, 50, WIDTH - 10, 90, "black", 1))
-        url = str(self.tabs[self.active_tab].url)
-        cmds.append(DrawText(55, 55, url, buttonfont, "black"))
+            url = str(self.browser.tabs[self.browser.active_tab].url)
+            cmds.append(DrawText(
+                left_bar,
+                top_bar,
+                url, self.font, "black"))
```

Comment:
What's with the indent? If we want to change stuff within a block I always put the first line of the block somewhere to notate that.